features/smoke_tests/sessions.feature

```diff
@@ -49,6 +49,7 @@ Scenario: Automated sessions send
     And the payload field "events.0.session.id" equals the stored value "automated_session_id"
     And the event "session.events.handled" equals 1
     And the event "session.events.unhandled" equals 0
+    And the event "severityReason.unhandledOverridden" is false
```

Comment:
Is there a reason to assert this here and none of the other smoke tests?

Reply:
Updated to assert this in other smoke tests